### DISCUSSION TITLE
Way to determinate animation status in ZeppOS 2.0

### DISCUSSION THREAD

**Discussion by zdunex25** [zepp-health/zeppos-forum#96] [Q&A] [answered]
So in case of OS 1.0 I had to use IMG_ANIM that supports:
_hmUI.prop.ANIM_IS_RUNINNG
hmUI.prop.ANIM_IS_PAUSE
hmUI.prop.ANIM_IS_STOP_

Now for OS 2.0 there is no way to know animation state (if you need to avoid using IMG_ANIM): https://docs.zepp.com/docs/reference/device-app-api/newAPI/ui/widgetAnimations/ 

**The problem:**
I initiate simple animation in my watch face everytime user goes to the main screen using:
`resume_call: (function () {
    myanimation();
}),`
and now I wanna avoid animation to start multiple times when user will (for example) swipe statusbar or notification page more than once.
From what I understand the best way to do so is to stop/reset animation state in pause_call, but how to do it when there is no status available?
@MyoungXUE @htoooth

**Comment by MyoungXUE** [comment, marked as answer]
Hello, I have confirmed with relevant colleagues and found that this part of the document is not perfect, we will add as soon as possible.

**Reply by zdunex25** [reply]
It's there now, thank you https://docs.zepp.com/docs/reference/revision-history/
